homeassistant/components/fan/__init__.py

```diff
@@ -140,6 +148,18 @@ def set_speed(hass, entity_id: str=None, speed: str=None) -> None:
     hass.services.call(DOMAIN, SERVICE_SET_SPEED, data)
 
 
+def set_drection(hass, entity_id: str=None, direction: str=None) -> None:
```

Comment:
Any new service will have to be implemented in the demo fan and also get tested.